addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -196,6 +207,11 @@ odoo.define('website.s_website_form', function (require) {
                 inputEl.disabled = !!this._disabledStates.get(inputEl);
             }
 
+            // Remove the fake input file button and the information text and put the real one back.
+            this.$target[0].querySelectorAll('input[class="fake-button mr-1"]').forEach(el => el.remove());
+            this.$target[0].querySelectorAll('label[class="file-info"]').forEach(el => el.remove());
```

Comment:
Try using the same kind of thing for these selectors. (e.g. `label.o_file_info`)